File: filedefs/filetypes.c
[build-menu]
FT_01_LB=_Build
FT_01_CM=gcc -Wall -Wextra -Wpedantic -O3 -o "%e" "%f" -lm && strip "%e"
FT_01_WD=
FT_00_LB=_Compile
FT_00_CM=gcc -Wall -Wextra -Wpedantic -c "%f"
FT_00_WD=
EX_01_LB=Stylise
EX_01_CM=indent -kr "%f"; astyle --style=kr --indent=spaces=4 -p "%f"; rm "%f.orig" "%f~"
EX_01_WD=

File: filedefs/filetypes.java
[build-menu]
FT_01_LB=Build
FT_01_CM=javac -Xlint:all "%f"
FT_01_WD=
FT_00_LB=Package
FT_00_CM=javac -Xlint:all -d . *.java
FT_00_WD=
EX_00_LB=_Execute
EX_00_CM=java "%e"
EX_00_WD=
EX_01_LB=Applet
EX_01_CM=appletviewer "%f"
EX_01_WD=
FT_02_LB=Stylise
FT_02_CM=astyle --style=java --indent=spaces=4 -p "%f"; rm "%f.orig"
FT_02_WD=
